packages/caip/src/adapters/coingecko/index.test.ts

```diff
@@ -124,7 +124,7 @@ describe('adapters:coingecko', () => {
       expect(coingeckoToAssetIds('cosmos')).toEqual([atomOnCosmos, atomOnBsc])
     })
 
-    it('can get AssetIds for USD Coin on EVM Chains', () => {
+    it('can get AssetIds for USD Coin on chains using coingecko', () => {
```

Comment:
```suggestion
    it('can get AssetIds for USD Coin on Solana and EVM chains using coingecko', () => {
```
Though can we make this its own test not to mix EVM/erc20 concerns and Solana?

Reply:
I think we can't because coingeckoToAssetIds will always return every usdc occurences

Reply:
Fixed in [ef1db6d](https://github.com/shapeshift/web/pull/7798/commits/ef1db6d6a7e416a0a0bcd7132926a700fb337b6c)